### ISSUE TITLE
a issue about method returnType

### ISSUE THREAD

**Issue by beeant0512** [mybatis/generator#109]
when i set a method returnType to `java.util.List<com.beeant.dto.User>` and i add a importedType to the interfaze,
here is the code
~~~
newMethod.setReturnType(methodReturn);
if (methodReturn.getShortName().indexOf("List") != -1){
    interfaze.addImportedType(methodReturn);
}
~~~

and the generate clientMethod will like this:
~~~
import com.beeant.dto.User;
import com.github.miemiedev.mybatis.paginator.domain.PageBounds;
import com.github.miemiedev.mybatis.paginator.domain.PageList;
import java.util.List;
~~~

~~~
java.util.List<com.beeant.dto.User> selectByExample(User user);
~~~

**Comment by jeffgbutler** [comment]
Thanks for reporting this.  I'll take a look at it.

**Comment by jeffgbutler** [comment]
Fixed via #112
